docs/development.rst

```diff
@@ -0,0 +1,283 @@
+
+This section is just for information. These packages will be installed as part
+of the steps described in :ref:`Setting up the development environment`.
+
++--------------------------+--------------------+----------------------+-------+
+| Op.system / Distribution | Package name       | Version requirements | Notes |
++==========================+====================+======================+=======+
+| Linux RedHat family      |                    |                      |       |
+| (RHEL, CentOS, Fedora)   |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| Linux Debian family      |                    |                      |       |
+| (Ubuntu, Debian,         |                    |                      |       |
+| LinuxMint)               |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| Linux SUSE family        |                    |                      |       |
+| (SLES, openSUSE)         |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| OS-X                     |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+| Windows                  |                    |                      |       |
++--------------------------+--------------------+----------------------+-------+
+
+
+.. _`Building the documentation`:
+
+Building the documentation
+--------------------------
+
+The ReadTheDocs (RTD) site is used to publish the documentation for the
+pywbem package at https://pywbem.readthedocs.io/
+
+This page is automatically updated whenever the Git repo for this package
+changes the branch from which this documentation is built.
+
+In order to build the documentation locally from the Git work directory,
+execute:
+
+::
+
+    $ make builddoc
+
+The top-level document to open with a web browser will be
+``build_doc/html/docs/index.html``.
+
+
+.. _`Testing`:
+
+.. # Keep the tests/README file in sync with this 'Testing' section.
+
+Testing
+-------
+
+
+All of the following `make` commands run the tests in the currently active
+Python environment. Depending on how the `pywbem` package is installed in that
+Python environment, either the `pywbem` and `pywbem_mock` directories in the
+main repository directory are used, or the installed `pywbem` package.
+The test case files and any utility functions they use are always used from
+the `tests` directory in the main repository directory.
+
+The `tests` directory has the following subdirectory structure:
+
+::
+
+    tests
+     +-- unit                Unit tests
+     |    +-- utils               Utility functions used by unit tests
+     +-- manual              Manual tests
+     +-- schema              The CIM schema MOF files used by some MOF tests
```

Comment:
The `manual` directory is still named `live_unit` in the repo. Change it in the repo.

Reply:
DONE in pr # 230, committed.